dipy/viz/actor.py

```diff
@@ -1337,6 +1339,79 @@ def point(points, colors, opacity=1., point_radius=0.1, theta=8, phi=8):
     return actor
 
 
+def sphere(centers, colors, radii=1., theta=16, phi=16):
+    """ Visualize one or many spheres with different colors and radii
+
+    Parameters
+    ----------
+    centers : ndarray, shape (N, 3)
+    colors : ndarray (N,3) or (N, 4) or tuple (3,) or tuple (4,)
+        RGB or RGBA (for opacity) R, G, B and A should be at the range [0, 1]
+    radii : float or ndarray, shape (N,)
+    theta : int
+    phi : int
+
+    Returns
+    -------
+    vtkActor
+
+    Examples
+    --------
+    >>> from dipy.viz import window, actor
+    >>> ren = window.Renderer()
+    >>> centers = np.random.rand(5, 3)
+    >>> sphere_actor = actor.sphere(centers, window.colors.coral)
+    >>> ren.add(sphere_actor)
+    >>> #window.show(ren)
+    """
+
+    if np.array(colors).ndim == 1:
+        colors = np.tile(colors, (len(centers), 1))
+
+    if isinstance(radii, (float, int)):
+        radii = radii * np.ones(len(centers), dtype='f8')
+
+    pts = numpy_to_vtk_points(centers)
+    cols = numpy_to_vtk_colors(255 * colors)
+    cols.SetName('colors')
+
+    radii_fa = numpy_support.numpy_to_vtk(radii.astype('f8'), deep=0)
+    radii_fa.SetName('rad')
+
+    src = vtk.vtkSphereSource()
+    src.SetRadius(0.5)
+    src.SetThetaResolution(theta)
+    src.SetPhiResolution(phi)
+
+    polyData = vtk.vtkPolyData()
+    polyData.SetPoints(pts)
+    polyData.GetPointData().AddArray(radii_fa)
+    polyData.GetPointData().SetActiveScalars('rad')
+    polyData.GetPointData().AddArray(cols)
+
+    glyph = vtk.vtkGlyph3D()
+    glyph.SetSourceConnection(src.GetOutputPort())
+    if major_version <= 5:
```

Comment:
As mentioned by @nilgoyette in #1522, we should decide on an alternate name for this variable. Like `vtk_major_version`.

Reply:
Sure but this is a topic for a different PR and see here https://github.com/nipy/dipy/issues/1544 Support of old vtk versions should be removed all together. More of a problem than a solution.